packages/opam-ed/opam-ed.0.1/opam

```diff
@@ -0,0 +1,14 @@
+opam-version: "1.2"
+maintainer: "Louis Gesbert <louis.gesbert@ocamlpro.com>"
+authors: "Louis Gesbert <louis.gesbert@ocamlpro.com>"
+homepage: "https://github.com/AltGr/opam-ed"
+bug-reports: "https://github.com/AltGr/opam-ed/issues"
+license: "GPL-2.1 with OCaml linking exception"
+dev-repo: "git+https://github.com/AltGr/opam-ed.git"
+build: [ make "COMP=ocamlc" {!ocaml-native} ]
+depends: [
+  "ocamlfind"
+  "cmdliner"
+  "opam-file-format" {>= "2.0.0~beta3"}
+]
+available: [ocaml-version >= "4.03.0"]
```

Comment:
I considered it, but it doesn't seem needed: it's called `opam-*` but I don't think you would want to install it automatically, and it's not really tied to opam in terms of functionality.